randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -258,6 +258,7 @@ Extra - asset_id: collision_camera_012
   > Door to Spider Ball Balcony
       Trivial
   > Door to Outer Hub
+      # SWJ: https://youtu.be/n_gpWjN5Px0
```

Comment:
FWIW, this should also be rerecorded with inputs at a later date. If you want you *can* comment that in there, but I dont mind if it isnt.

Reply:
Ill leave it as is for this PR, i do plan on re-recording all videos with input display since ive got it just setup through OBS permanently now.